store/src/main/java/in/testpress/store/PaymentGatewayFactory.kt

```diff
@@ -3,9 +3,16 @@ package `in`.testpress.store
 import `in`.testpress.store.models.Order
 import `in`.testpress.store.payu.PayuPaymentGateway
 import android.app.Activity
+import `in`.testpress.store.razorpay.RazorpayPaymentGateway
+import `in`.testpress.core.TestpressSdk
+import `in`.testpress.models.InstituteSettings
+
 
 class PaymentGatewayFactory {
     fun create(order: Order, activity: Activity): PaymentGateway {
+        val instituteSettings: InstituteSettings = TestpressSdk.getTestpressSession(activity)!!.instituteSettings
+        if (instituteSettings.currentPaymentApp.lowercase() == "razorpay")
```

Comment:
My doubt is, where do I set this currentPaymentApp variable ? For testing purpose while developing I had commented out `PayuPaymentGateway` and had directly used `RazorpayPaymentGateway`.
But will this variable currentPaymentApp come dynamically I am not sure.